packages/atomic/cypress/integration/facets/category-facet.cypress.ts

```diff
@@ -0,0 +1,456 @@
+import {
+  setUpPage,
+  shouldRenderErrorComponent,
+} from '../../utils/setupComponent';
+import {
+  FacetSelectors,
+  createAliasShadow,
+  createAliasFacetUL,
+  FacetAlias,
+  BreadcrumbSelectors,
+} from './facet-selectors';
+import {
+  validateFacetComponentLoaded,
+  validateFacetNumberofValueEqual,
+  assertNonZeroFacetCount,
+  assertClickShowMore,
+  assertShowMoreUA,
+  assertClickShowLess,
+  assertShowLessUA,
+} from './facet-utils';
+import {
+  convertArrayToString,
+  doSortAlphanumeric,
+} from '../../utils/componentUtils';
+
+const categoryFacetProp = {
+  field: 'geographicalhierarchy',
+  label: 'Atlas',
+  defaultCategoryLabel: 'All Categories',
+  numberOfValue: 5,
+  numberOfFacetValueAfterExpand: 7,
+};
+
+const canadaHierarchy = ['North America', 'Canada', 'Alberta'];
+function setupCategoryFacet(field: string, label: string, option?: {}) {
+  setUpPage(`
+    <atomic-breadcrumb-manager></atomic-breadcrumb-manager>    
+    <atomic-category-facet field="${field}" label="${label}" ${option}></atomic-category-facet>`);
+}
+
+function assertClearAllTitleAndTotalParents(
+  clearAllTitle: string,
+  totalParents: number
+) {
+  cy.get(FacetAlias.facetShadow)
+    .find(FacetSelectors.categoryFacetClearLevelButton)
+    .should('contain.text', clearAllTitle);
+  cy.get(FacetAlias.facetShadow)
+    .find('ul[part="parents"] li')
+    .its('length')
+    .should('eq', totalParents);
+}
+
+function assertTotalChildrentMoreThan(value: number) {
+  cy.get(FacetAlias.facetShadow)
+    .find('ul')
+    .not('[part="parents"]')
+    .find('li')
+    .its('length')
+    .should('be.gt', value);
+}
+
+function clickOnCategoryFacetWithValue(value: string) {
+  cy.get(FacetAlias.facetUL)
+    .find(FacetSelectors.categoryFacetNextLevelButton)
+    .contains(value)
+    .click();
+}
+
+describe('Category Facet with default setting', () => {
+  beforeEach(() => {
+    setupCategoryFacet(categoryFacetProp.field, categoryFacetProp.label);
+    createAliasShadow(categoryFacetProp.field, FacetSelectors.categoryFacet);
+    createAliasFacetUL(categoryFacetProp.field, FacetSelectors.categoryFacet);
```

Comment:
can AliasShadow and AliasFacetUL be merged?

Reply:
unfortunately no, the Label, searchbox, and clearAll button are not under the `ul`